### ISSUE TITLE
Editor: Accessibility Issues with labels

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4454]
**Describe the bug**
Editor has some labeling issues.

**To Reproduce**
**Issue One**
The following form fields do not provide a programmatically associated label that describes the purpose of each field:
Note field (rich text editor) in the Take Note dialog modal.
Although a <label> element is correctly associated with the field, the field is a <div> element with role="textbox" rather than a native HTML <textarea> element or <input type="text">. As a result, the label relationship is not valid.

```
<p id="note-label">Note</p>
<div role="textbox" aria-labelledby="note-label"...></div>
```

**Issue Two**
The Font style input does not have a valid label, but instead relies on the current value of the input to convey its purpose. By default the control is announced as "Normal Text".
```
<button type="button" class="btn fontpicker hide-focus is-open" data-action="fontStyle" aria-haspopup="true" aria-controls="popupmenu-4" tabindex="0" style="width: 128px;">...
```
To fix add a label in the button.
**Expected behavior**
Uses correct aria.

**Version**
- ids-enterprise: [v4.33]

**Platform**
 - Chrome/Any

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-31955

**Comment by ericangeles** [comment]
@tmcconechy where does these issues found?

**Comment by tmcconechy** [comment]
@ericangeles you can see these both on the main editor page i think https://master-enterprise.demo.design.infor.com/components/editor/example-index.html

**Comment by tmcconechy** [comment]
Seems good. Just a note i was concerned about if there was multiple editors in the page like this

```

    <div class="field">
      <span id="editor1-label" class="label">Comments</span>
      <div class="editor" id="editor1" aria-label="Comments" data-options="{attributes: [{name: 'id', value: 'example1'}, {name: 'data-automation-id', value: 'automation-id-example1'}]}">
        <p>Embrace <a href="http://en.wikipedia.org/wiki/e-commerce" class="hyperlink">e-commerce action-items</a>, reintermediate, ecologies paradigms wireless share life-hacks create innovative harness. Evolve solutions rich-clientAPIs synergies harness relationships virtual vertical facilitate end-to-end, wireless, evolve synergistic synergies.</p>
        <p>Cross-platform, evolve, ROI scale cultivate eyeballs addelivery, e-services content cross-platform leverage extensible viral incentivize integrateAJAX-enabled sticky evolve magnetic cultivate leverage; cutting-edge. Innovate, end-to-end podcasting, whiteboard streamline e-business social; compelling, "cross-media exploit infomediaries innovative integrate integrateAJAX-enabled." Killer interactive reinvent, cultivate widgets leverage morph.</p>
      </div>
    </div>

    <div class="field">
      <span id="editor2-label" class="label">Comments</span>
      <div class="editor" id="editor1" aria-label="Comments" data-options="{attributes: [{name: 'id', value: 'example1'}, {name: 'data-automation-id', value: 'automation-id-example1'}]}">
        <p>Embrace <a href="http://en.wikipedia.org/wiki/e-commerce" class="hyperlink">e-commerce action-items</a>, reintermediate, ecologies paradigms wireless share life-hacks create innovative harness. Evolve solutions rich-clientAPIs synergies harness relationships virtual vertical facilitate end-to-end, wireless, evolve synergistic synergies.</p>
        <p>Cross-platform, evolve, ROI scale cultivate eyeballs addelivery, e-services content cross-platform leverage extensible viral incentivize integrateAJAX-enabled sticky evolve magnetic cultivate leverage; cutting-edge. Innovate, end-to-end podcasting, whiteboard streamline e-business social; compelling, "cross-media exploit infomediaries innovative integrate integrateAJAX-enabled." Killer interactive reinvent, cultivate widgets leverage morph.</p>
      </div>
    </div>
```

But this worked the way you did it 👍

**Comment by jbrcna** [comment]
**QA Passed**
it says _"Font Style Normal Text Pop up Button"_
v4.35.0-dev
https://master-enterprise.demo.design.infor.com/components/editor/example-index.html
![image](https://user-images.githubusercontent.com/56722184/98800979-1d24d680-244c-11eb-917d-2f19dabc3a4a.png)
![image](https://user-images.githubusercontent.com/56722184/98801343-a50ae080-244c-11eb-9522-7ecee1420b9f.png)
